### PR TITLE
Update CONTRIBUTING.rst

### PR DESCRIPTION
**Pull request by fehrhart**
Adding an example how to add a new sparql query to scholia

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -67,3 +67,39 @@ Before committing please run the following code in the main directory:
...
+
+Examples
+-------
+
+Adding new SPARQL queries to Scholia
+
+1. Assign the task for yourself (if it is in an issue tracker)
+2. Get the new query e.g. 
+E.g. 
+SELECT DISTINCT ?author ?authorLabel ?award ?awardLabel WHERE {
```
Comment (fnielsen) [inline CONTRIBUTING.rst]: There is a bit of a formatting issue with the code. Perhaps some indentation is needed?

Comment (fehrhart) [reply CONTRIBUTING.rst]: looks not as intended... ill check!

**Comment by fehrhart** [comment]
I think I got it now... 
Someone please check if code:: python is correct for line 91! 
Please give the correct file name for line 89

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -77,15 +77,18 @@ Adding new SPARQL queries to Scholia
...
            wdt:P50 ?author .
        ?author wdt:P166 ?award .
    SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }  
    }
 
-3. Add a name (within your own fork of the file https://github.com/fnielsen/scholia/tree/master/scholia[…])
-e.g. authorAwardsSparql
+3. Add a name (within your own fork of the file https://github.com/fnielsen/scholia/tree/master/scholia[…]) e.g. authorAwardsSparql
+
+.. code:: python
```
Comment (fnielsen) [inline CONTRIBUTING.rst]: This should be Javascript

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -97,10 +100,14 @@ e.g. authorAwardsSparql
 
 4. Add that name to the sparql-to-data table
 
+.. code:: python
```
Comment (fnielsen) [inline CONTRIBUTING.rst]: This is also Javascript

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -97,10 +100,14 @@ e.g. authorAwardsSparql
...
+
    sparqlToDataTable(authorAwardsSparql, "#author-awards");
 
 5. Add some table formatting
 
+.. code:: html
+
    <h2 id="AuthorAwards">Author Awards</h2>
```
Comment (fnielsen) [inline CONTRIBUTING.rst]: I think `<h2 id="Author-awards">Author awards</h2>` would be better, i.e., lower case section words, except the first. A make a dash in the id for the space.

File: CONTRIBUTING.rst [hunk 5]
```diff
@@ -77,15 +77,18 @@ Adding new SPARQL queries to Scholia
...
    SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }  
    }
 
-3. Add a name (within your own fork of the file https://github.com/fnielsen/scholia/tree/master/scholia[…])
-e.g. authorAwardsSparql
+3. Add a name (within your own fork of the file https://github.com/fnielsen/scholia/tree/master/scholia[…]) e.g. authorAwardsSparql
+
+.. code:: python
 
    authorAwardsSparql =
```
Comment (fnielsen) [inline CONTRIBUTING.rst]: There is a backtick missing

File: CONTRIBUTING.rst (hunk 2 above)
Comment (fehrhart) [reply CONTRIBUTING.rst]: done

File: CONTRIBUTING.rst (hunk 3 above)
Comment (fehrhart) [reply CONTRIBUTING.rst]: done

File: CONTRIBUTING.rst (hunk 4 above)
Comment (fehrhart) [reply CONTRIBUTING.rst]: done

File: CONTRIBUTING.rst (hunk 5 above)
Comment (fehrhart) [reply CONTRIBUTING.rst]: done

**Comment by fnielsen** [comment]
Here are a few issues you might consider
1. Fix identation in SPARQL code (SERVICE lines and  "wdt:P50 ?author ." for instance)
2. Change "Author Awards" to "Author awards"
3. Add a "`"'s to the Javascription code.

**Comment by fnielsen** [comment]
I have just done the issues. I hope they are ok?

**Comment by fehrhart** [comment]
sure, thanks!

**Pull request merged**